examples/analysis/plot_CNAP.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3.7
```

Comment:
These lines should be removed so that linux based doesnt switch to using python 3.7. This line is not required for files ending in .py

Reply:
Okie, but that line exists in every examples/analysis .py file. Remove from all?